app/src/main/res/values/preference_keys.xml

```diff
@@ -11,7 +11,7 @@
     <string name="pref_key_privacy_block_social" translatable="false"><xliff:g id="preference_key">pref_privacy_block_social</xliff:g></string>
     <string name="pref_key_privacy_block_cryptomining" translatable="false"><xliff:g id="preference_key">pref_privacy_block_cryptomining</xliff:g></string>
     <string name="pref_key_privacy_block_fingerprinting" translatable="false"><xliff:g id="preference_key">pref_privacy_block_fingerprinting</xliff:g></string>
-    <string name="pref_key_privacy_block_other" translatable="false"><xliff:g id="preference_key">pref_privacy_block_other</xliff:g></string>
+    <string name="pref_key_privacy_block_other2" translatable="false"><xliff:g id="preference_key">pr_ef_privacy_block_other2</xliff:g></string>
```

Comment:
Ahhh, typo!

Reply:
Fixed in [v91.2.2](https://github.com/mozilla-mobile/focus-android/releases/tag/v91.2.2) and backported.


😰